lib/ansible/config/base.yml

```diff
@@ -405,7 +405,7 @@ DEFAULT_ASK_SU_PASS:
   default: False
   deprecated:
     why: In favor of Ansible Become, which is a generic framework. See become_ask_pass.
-    version: "2.8"
+    version: "2.9"
```

Comment:
Is this part of a larger conversation about the schedule for deprecating config options? I'm curious about why some are getting moved from 2.8 to 2.9, while others are getting moved from 2.8 to 2.12. If there's an issue somewhere that documents those decisions, a link to it would be helpful for context.